pom.xml

```diff
@@ -50,7 +50,8 @@
     <infinispan.version>7.1.0.CR1</infinispan.version>
     <openjpa.jdbc.version>2.1.1</openjpa.jdbc.version>
     <mapkeeper.version>1.0</mapkeeper.version>
-    <mongodb.version>2.11.2</mongodb.version>
+    <mongodb.version>2.12.2</mongodb.version>
```

Comment:
can we update this to a more recent version? I know #221 was trying to go to 2.13.0